platform/netbinox/src/org/netbeans/modules/netbinox/JarBundleFile.java

```diff
@@ -49,14 +54,31 @@
  * @author Jaroslav Tulach <jtulach@netbeans.org>
  */
 final class JarBundleFile extends BundleFile implements BundleContent {
```

Comment:
would be good to mention in a comment that this class shares code with `JarClassLoader` and vice versa, so that future readers know that they have to mirror changes.